cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
This file is not useful, we should remove it. If it's documentation, it belongs in a README